### ISSUE TITLE
Invalid .stylelintrc JSON throws: "SyntaxError: Unexpected string"

### ISSUE THREAD

**Issue by MadLittleMods** [AtomLinter/linter-stylelint#12]
When using a custom `.stylelintrc`(uncheck `Use Preset`) that has some JSON syntax errors, the following error is thrown which is not immediately clear what the error actually is. Adding a nicer message explaining what went wrong could help ease confusion.

`linter-stylelint@v1.0.7`

```
SyntaxError: Unexpected string
  at Object.parse (native)
  at Object.lint (/Users/Eastwood/.atom/packages/linter-stylelint/index.js:76:25)
  at Promise.then._this.emitter.emit.linter (/Users/Eastwood/.atom/packages/linter/lib/linter-registry.coffee:59:22)
  at LinterRegistry.triggerLinter (/Users/Eastwood/.atom/packages/linter/lib/linter-registry.coffee:58:16)
  at /Users/Eastwood/.atom/packages/linter/lib/linter-registry.coffee:52:17
  at Array.map (native)
  at /Users/Eastwood/.atom/packages/linter/lib/linter-registry.coffee:50:27
```

**Comment by ismay** [comment]
I'm getting this error as well. Don't know if it should fail silently, but it now shows the error on every save, which is a bit much.

**Comment by 1000ch** [comment]
@MadLittleMods @ismay Thank you for reporting. Fixed in `v1.1.1`.
